packages/jdk11.rb

```diff
@@ -1,46 +1,85 @@
 require 'package'
+require 'uri'
 
 class Jdk11 < Package
-  description 'The JDK is a development environment for building applications, applets, and components using the Java programming language.'
-  homepage 'https://www.oracle.com/java/technologies/javase-jdk11-downloads.html'
+  description 'The Oracle JDK is a development environment for building applications, applets, and components using the Java programming language.'
+  homepage 'https://www.oracle.com/java'
   version '11.0.16.1'
```

Comment:
```
@_ver = '11.0.16.1'
version "#{@_ver}-1"
```
Replace all instances of `#{version}` with `#{@_ver}` since we need to bump the version?

Reply:
Done